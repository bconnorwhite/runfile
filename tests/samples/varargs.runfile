# ----------
# Varargs Support
# ----------

# Simple varargs
test ...args:
  cargo test $args

# Varargs with other arguments
run
  target
  ...args:
  cargo run $target $args

# Varargs with flags
build
  --release
  ...args:
  cargo build $release $args
