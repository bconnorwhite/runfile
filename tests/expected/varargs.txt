Varargs Support
  test          # Simple varargs
    ...args
  run           # Varargs with other arguments
    target
    ...args
  build         # Varargs with flags
    ...args
    --release
